core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -455,6 +482,19 @@ class StandaloneDynamicAllocationSuite
     sc.killExecutors(getExecutorIds(sc).take(n))
   }
 
+  private def killExecutorWithForce(
+      sc: SparkContext,
+      executorId: String,
+      force: Boolean = true): Boolean = {
```

Comment:
I would just call this `killExecutor` since `force` can be false. Also to make things clearer I wouldn't make this take in a default value.